blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueRun.java

```diff
@@ -186,13 +186,18 @@
 
     /**
      *
-     * @return Gives Actions associated with this Run
+     * @return Gives Actions associated with this Run, if requested via tree
      */
     @Navigable
     @Exported(name = ACTIONS, inline = true)
     public abstract Collection<BlueActionProxy> getActions();
 
     /**
+     * @return All Actions for this Run
+     */
+    public abstract Collection<BlueActionProxy> getAllActions();
```

Comment:
I needed to add this method because the implementation of `getActions` checks the tree API params and doesn't return anything unless that param is specified.

Reply:
Why are actions needed? Reason we do not return actions (without tree param) because serializing all actions could be very expensive. 🐛

Reply:
I will remove this method as part of this PR, as we are splitting out the Coverage / Jacoco trend into its own plugin. Originally I added this method because the `JacocoBuildAction` contained the coverage data from the Run. Is there some other way to get at this same information? I didn't see it.

Reply:
Also to clarify further, `getAllActions` was not annotated with `@Exported` so it would not go over the wire.

Reply:
@cliffmeyers To look for specific action you can do something like: `MultiBranchProject.getAction(JacocoBuildAction.class)`.